api/src/main/resources/_core/database/mysql/dimensions/dim_metadata/sp_mamba_dim_concept_metadata_update.sql

```diff
@@ -1,5 +1,8 @@
 -- $BEGIN
 
+-- Update/insert table column metadata configs without table_columns json
+CALL sp_mamba_missing_table_column_names_insert();
```

Comment:
@smallgod Why are you calling a SP inside another SP that is not the `sp_processing` ? This is an update SP and you added a insert SP... Can't you just make sure the insert is called on the same file the update is being called but force the correct order?

Reply:
@ebambo I have moved the SP out. Though I had initially thought it would b okay since this is technically an update to that table. 
But yes, I have pushed a new commit with the change.
Thanks